### PR TITLE
Change maximum length of title field

### PR DESCRIPTION
**Pull request by HoshkoS**
## Code reviewers

@loqimean 
@AndriyAndriyovuch 

## Summary of change

Changed maximum length of title field to 30

**Before**
<kbd>image</kbd>

**After**
<kbd>image</kbd>

<kbd>image</kbd>

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by AndriyAndriyovuch — APPROVED** [review]

File: spec/models/site_setting_spec.rb [hunk 1]
```diff
@@ -30,8 +30,8 @@
       expect(site_setting).not_to be_valid
     end
 
-    it "is not valid with a title longer than 20 characters" do
-      site_setting.title = "a" * 21
+    it "is not valid with a title longer than 30 characters" do
+      site_setting.title = "a" * 31
       expect(site_setting).not_to be_valid
     end
```
Comment (loqimean) [inline spec/models/site_setting_spec.rb:36]: shoulda_matchers

File: spec/models/site_setting_spec.rb [hunk 2]
```diff
@@ -12,48 +12,46 @@
...
-      site_setting.favicon.content_type = "application/pdf"
-      expect(site_setting).not_to be_valid
+      subject.favicon.content_type = "application/pdf"
+      is_expected.not_to be_valid
     end
 
     it "is valid with a favicon of valid type" do
-      site_setting.favicon.content_type = "image/png"
-      expect(site_setting).to be_valid
+      subject.favicon.content_type = "image/png"
```
[the remark this answers is not in the document]
Comment (loqimean) [reply spec/models/site_setting_spec.rb:47]: like `invalid_format_picture`

File: spec/models/site_setting_spec.rb [hunk 3]
```diff
@@ -12,48 +12,46 @@
...
+      is_expected.to be_valid
     end
 
     it "is not valid with a favicon larger than 500 KB" do
-      site_setting.favicon.byte_size = 600.kilobytes
-      expect(site_setting).not_to be_valid
-      expect(site_setting.errors.messages[:favicon]).to include(I18n.t("account.site_settings.validations.size"))
+      subject.favicon.byte_size = 600.kilobytes
```
Comment (loqimean) [inline spec/models/site_setting_spec.rb:52]: let

File: spec/models/site_setting_spec.rb [hunk 4]
```diff
@@ -12,48 +12,69 @@
   end
 
   context "validates properly" do
-    before { site_setting.update(site_setting_params) }
+    before { subject.update(site_setting_params) }
+
+    subject { described_class.current }
```
Comment (loqimean) [inline spec/models/site_setting_spec.rb:17]: поміняй місцями, а то не дуже читабельно виглядає)

**Review by loqimean — APPROVED** [review]

**Pull request merged**